src/elements/fields/SignatureField.jsx

```diff
@@ -34,7 +36,9 @@ function SignatureField({
         ref={(ref) => {
           signatureRef[servar.key] = ref;
         }}
-        onEnd={onEnd}
+        onEnd={() => {
+          onEnd();
```

Comment:
should I revert this to onEnd={onEnd} now, or do you prefer it like this to make sure the event isn't passed up?

Reply:
yeah onEnd={onEnd} would probably be best, it shouldn't matter if the onEnd is ignoring the param anyways

Reply:
My thinking with this was that maybe it's better to discard the param right away, rather than rely on something further up the stack to do so